README.md

```diff
@@ -1,2 +1,17 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+Create a list of random words which the same length
+Choose a word from the list to be the key word
+Guessed word = false
+Attempts =0
+While Guessed word = false
```

Comment:
You seem to be using `guessed word` both as a boolean value (whether the word has been guessed) and as a string (the word that has been guessed). Make sure names are used consistently.